src/hb-ot-layout-gpos-table.hh

```diff
@@ -601,15 +600,10 @@ struct SinglePosFormat2
     if (unlikely (!c->check_assign (valueFormat, valFormat))) return;
     if (unlikely (!c->check_assign (valueCount, it.len ()))) return;
 
-    + it
-    | hb_map (hb_second)
-    | hb_apply ([=] (hb_array_t<const Value> val_iter)
-		{
-		  + val_iter
-		  | hb_apply ([=] (const Value& _) { c->copy (_); })
-		  ;
-		})
-    ;
+    for (hb_array_t<const Value> val_iter : + it
+					    | hb_map (hb_second))
+      for (const Value& _ : val_iter)
+	c->copy (_);
```

Comment:
This part should be improved (I know old code did as is).  I wonder if `c->copy (val_iter)` would work.  We should make it work.

Reply:
Correct. Am looking into it.

Reply:
#1987